### DISCUSSION TITLE
How to provide integration with playwright

### DISCUSSION THREAD

**Discussion by tobiasBora** [vitest-dev/vitest#5892] [Q&A]
Hey,

In the very exciting annoucement of browser support in https://github.com/vitest-dev/vitest/discussions/5828 they mention that browser support is meant wnly for component testing and that e2e tests are supposed to be done using the existing playwright integration. But I can't find any documentation on this, how is this supposed to work?

Ideally I'd love to:
- get merged coverage and tests between vitests unit tests and playwright
- still be able to use playwright UI to debug tests (i.e. I'd love to have a format compatible with playwright)

And bonus if I can run playwright tests directly from vitests as well to have all tests listed in a single interface. Would be cool also to be able to enable/disable hot re-runs for some tests as they may take more time.

**Comment by CHC383** [comment]
Hi @sheremet-va, not sure if you have visibility on this question, so just tagging you to see if you have any insights. Thanks in advance!
